maven-failsafe-plugin/src/main/java/org/apache/maven/plugin/failsafe/IntegrationTestMojo.java

```diff
@@ -34,10 +34,11 @@
 import org.apache.maven.plugins.annotations.Mojo;
 import org.apache.maven.plugins.annotations.Parameter;
 import org.apache.maven.plugins.annotations.ResolutionScope;
-import org.apache.maven.shared.utils.ReaderFactory;
 import org.apache.maven.shared.utils.StringUtils;
 import org.apache.maven.surefire.suite.RunResult;
 
+import static org.apache.maven.shared.utils.ReaderFactory.FILE_ENCODING;
+
```

Comment:
I use it for several purposes. When line is getting to lone around 120 characters and the constant is just a storage, or assertion statement when it is ugly to read the class; otherwise the whole statement is not verbal enough if class appears. I use IntelliJ IDEA which navigates me to the class where the constant is stored. I do not know how bout Eclipse abilities. I tried to use same name of method in static import and Java compiler failed, so it's safe compiler.